rust_pajamax_bench/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM rust:1.82
+
+WORKDIR /app
+COPY rust_pajamax_bench /app
+COPY proto /app/proto
+
+RUN apt-get update && apt-get install -y cmake protobuf-compiler-grpc
+RUN rustup component add rustfmt
```

Comment:
```suggestion
```
This shouldn't be a part of the image.

Reply:
I copy this from [rust_tonic_mt_bench](https://github.com/LesnyRumcajs/grpc_bench/blob/master/rust_tonic_mt_bench/Dockerfile) .

Reply:
Well, it shouldn't be there as well. :)

Reply:
Ok, I remove this line.

Reply:
Done.

Reply:
Thanks!